src/lib/components/DataTable.svelte

```diff
@@ -447,14 +447,28 @@
     }
   }
 
+  export function getTablePagination() {
+    return {
+      currentPage: pagination.currentPage,
+      rowsPerPage: pagination.rowsPerPage,
+      totalRows,
+    }
+  }
+
+  export function changePagination(pag: {currentPage?: number, rowsPerPage?: number}) {
+    pag.currentPage? pagination.currentPage = pag.currentPage : null
```

Comment:
pagination.currentPage = pag?.currentPage
pagination.rowsPerPage = pag?.rowsPerPage

Reply:
pag.currentPage kan in dit scenario undefined zijn --> in de code die u voorstelt zal je pagination.currentPage undefined maken wat de paginatie zal breken?

Reply:
de ternary operator (condition ? true : false) wordt vooral gebruikt om een waarde te zetten
```js
let a_equals_b = ('a' = 'b') ? true : false
```
best niet gebruiken om logica in te stoppen, omdat dit dan niet leesbaar is
dus best:
```js
if (pag.currentPage) pagination.currentPage = pag.currentPage
```